lib/test-collection.js

```diff
@@ -68,6 +68,10 @@ TestCollection.prototype.add = function (test) {
 		return;
 	}
 
+	if (typeof test.fn === 'undefined') {
+		throw new Error('Test must be specified');
+	}
+
```

Comment:
Maybe this check would be better suited in [lib/runner.js#L52-L77](https://github.com/sindresorhus/ava/blob/e697629a77b511e069eb0a2991c8ee72cd09034c/lib/runner.js#L52-L77)